libraries/rush-lib/src/logic/operations/ShardedPhaseOperationPlugin.ts

```diff
@@ -137,7 +137,8 @@ function spliceShards(existingOperations: Set<Operation>, context: ICreateOperat
         displayName: collatorDisplayName,
         rushConfiguration,
         commandToRun,
-        customParameterValues: collatorParameters
+        customParameterValues: collatorParameters,
+        ignoredParameterNames: []
```

Comment:
These should be derived from invoking `getCustomParameterValuesForOperation()` on the operation that is being replaced with shards.